README.md

```diff
@@ -1,12 +1,12 @@
-# Bing Ads 
+# Microsoft Advertising 
```

Comment:
I know the name of the repo will be `dbt_microsoft_ads` for brevity's sake, but I think we should use the correct name whenever possible.